PeriodogramAnalysis/test_spectrogram_analysis.py

```diff
@@ -5,20 +5,7 @@
 from scipy.signal import spectrogram
 from miv.core.datatype import Signal
 from spectrogram_analysis import SpectrogramAnalysis
-
-
-def signal_input():
-    timestamps = np.linspace(0, 10, 1000, endpoint=False)
-    signal = (
-        np.sin(2 * np.pi * 5 * timestamps)
-        + np.sin(2 * np.pi * 10 * timestamps)
-        + np.sin(2 * np.pi * 15 * timestamps)
-        + np.sin(2 * np.pi * 20 * timestamps)
-    )
-
-    data1 = np.array([signal, signal]).T
-    signal = Signal(data=data1, timestamps=timestamps, rate=100)
-    return signal
+from test_power_density_statistics import mock_signal_generator
```

Comment:
https://docs.pytest.org/en/stable/how-to/fixtures.html#scope-sharing-fixtures-across-classes-modules-packages-or-session

Reply:
Good to know, I have implemented it in commit [9e1fd9b]